### DISCUSSION TITLE
Can Bark be fine-tuned?

### DISCUSSION THREAD

**Discussion by vineet-sangewar** [serp-ai/bark-with-voice-clone#93] [General]
I intend to use Bark for a use-case that involves some TV show content. I wanted to test if giving sample audios can help bark perform better.

**Comment by devinschumacher** [comment]
> I intend to use Bark for a use-case that involves some TV show content. I wanted to test if giving sample audios can help bark perform better.

We just launched our follow up to BARK -- an AI Voice Cloner Desktop App (runs locally, offline, super realistic voice cloning, data privacy, no coding needed)

👉Check it out: https://www.youtube.com/watch?v=84JVGtkapik
